src/clang.rs

```diff
@@ -448,7 +453,11 @@ impl Cursor {
 
     /// Get the visibility of this cursor's referent.
     pub fn visibility(&self) -> CXVisibilityKind {
-        unsafe { clang_getCursorVisibility(self.x) }
+        if clang_getCursorVisibility::is_loaded() {
+            unsafe { clang_getCursorVisibility(self.x) }
+        } else {
+            CXVisibility_Default
```

Comment:
There's a lot of failures:

https://gist.github.com/pornel/ffee1e2a0448ea37accc15e34ad90088

but it worked enough to get libvpx running :)